### PR TITLE
Cookie expiration period in load balancer policy

### LINKED ISSUE #126: Implement "CookieExpirationPeriod" parameter in outscale_load_balancer_policy
### Current Terraform Version

```
Terraform v1.1.8
+ provider v0.5.3
```

### Use-cases

```terraform
resource "outscale_load_balancer" "load_balancer02" {
  load_balancer_name = "terraform-load-balancer-for-app-policy"
  subregion_names    = ["${var.region}b"]
  listeners {
      load_balancer_port     = 80
      load_balancer_protocol = "TCP"
      backend_port           = 80
      backend_protocol       = "TCP"
  }
}

resource "outscale_load_balancer_policy" "load_balancer_policy02" {
  CookieExpirationPeriod = 2
  load_balancer_name = outscale_load_balancer.load_balancer02.load_balancer_name
  policy_name        = "terraform-load-balancer-app-policy"
  policy_type        = "app"
  cookie_name        = "cookie01"
  depends_on         = [outscale_load_balancer.load_balancer02]
}

```
```
terrafrom init
terraform apply
```

### PR DESCRIPTION
**Pull request by FredericBerot-Armand**
closes #126

### CODE REVIEW

File: outscale/resource_outscale_load_balancer_policy.go [hunk 1]
```diff
@@ -298,6 +312,9 @@ func resourceOutscaleAppCookieStickinessPolicyCreate(d *schema.ResourceData, met
 				a := make(map[string]interface{})
 				a["cookie_name"] = v.CookieName
 				a["policy_name"] = v.PolicyName
+				if v.GetPolicyName() == pn.(string) {
+					d.Set("cookie_name", v.GetCookieName())
+				}
```
Comment (outscale-toa) [inline outscale/resource_outscale_load_balancer_policy.go:325]: I don't think this condition is useful

File: outscale/resource_outscale_load_balancer_policy.go [hunk 2]
```diff
@@ -308,6 +325,9 @@ func resourceOutscaleAppCookieStickinessPolicyCreate(d *schema.ResourceData, met
 			for k, v := range *lb.LoadBalancerStickyCookiePolicies {
 				a := make(map[string]interface{})
 				a["policy_name"] = v.PolicyName
+				if v.GetPolicyName() == pn.(string) {
+					d.Set("cookie_expiration_period", cast.ToInt32(v.CookieExpirationPeriod))
+				}
```
Comment (outscale-toa) [inline outscale/resource_outscale_load_balancer_policy.go:338]: Same here

**Review by outscale-toa — CHANGES_REQUESTED** [review]

File: outscale/resource_outscale_load_balancer_policy.go (hunk 1 above)
Comment (FredericBerot-Armand) [reply outscale/resource_outscale_load_balancer_policy.go:325]: It's useful if you have multiples Policies, to update the state with the right name/period

File: outscale/resource_outscale_load_balancer_policy_test.go [hunk 3]
```diff
@@ -108,9 +108,9 @@ resource "outscale_load_balancer" "lb" {
...
 resource "outscale_load_balancer_policy" "foo" {
-	policy_type = "app"
+	policy_type = "load_balancer"
 	policy_name = "foo-policy"
 	load_balancer_name = "${outscale_load_balancer.lb.id}"
-	cookie_name = "MyOtherAppCookie"
-}`, rName, zone)
+	cookie_expiration_period = 180
+	}`, rName, zone)
 }
```
[the remark this answers is not in the document]
Comment (FredericBerot-Armand) [reply outscale/resource_outscale_load_balancer_policy_test.go:116]: Ok a change test

**Review by outscale-toa — APPROVED** [review]

**Pull request merged**